components/automate-deployment/pkg/server/helpers.go

```diff
@@ -0,0 +1,27 @@
+package server
+
+import (
+	"bytes"
+	"fmt"
+)
+
+func LogRotateConf(path string, params ...string) string {
```

Comment:
what if we pass empty params? something like 
LogRotateConf(" some path", "copytruncate", "")